share/man/man1/brew.1

```diff
@@ -554,10 +554,6 @@ HOMEBREW_BROWSER
 If set, uses this setting as the browser when opening project homepages, instead of the OS default browser\.
 .
 .TP
-HOMEBREW_BUILD_FROM_SOURCE
```

Comment:
> Building from source does not get you higher security, it gets you lower. We do not use a sandbox so you are executing arbitrary buildsystem code on your machine as an admin user. 

Security comes from many aspects. The code execution problem can be migrated by vm. What I mean here that building from source is more security than using third party binaries is focusing one of the aspects: there is no perfect way to verify whether a binary file is comprised. Even with reproducible bottles is not perfect. 

> The more options and entropy: the worse the user experience for the majority of users in favour of a small, vocal minority. That is a bad way of doing software. 

I think it's naive to think everyone have the same need and we should offer an unify UX for all them. What we should do is making the default UX to favor the majority need. At the same time, we should individuals custom their UX based on their own. And this is including (a) keep such feature unless it's too expensive to maintain (b) well document the optional features (which by nature means it would be used in only small group)